lib/services.js

```diff
@@ -254,7 +254,8 @@ export default {
                     return cb(err);
                 }
                 omVal.acl = parsedACL;
-                metadata.putObjectMD(bucketName, objectKey, omVal, log, err => {
+                metadata.putObjectMD(bucketName, objectKey, omVal, {}, log,
+                err => {
```

Comment:
Now all application using `putObjectMD` and `deleteObjectMD` should have the callback `callback(err, data)`.